app/src/main/java/org/standardnotes/notes/SettingsActivity.kt

```diff
@@ -32,8 +35,22 @@ class SettingsActivity : BaseActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    private fun exportData() {
+        val listener: ExportUtil.ExportListener = object : ExportUtil.ExportListener {
+            override fun onExportFailed() {
+                Snackbar.make(root, R.string.error_fail_export, Snackbar.LENGTH_SHORT).show()
+            }
+        }
+        if (radio_dec.isChecked) {
```

Comment:
you can just condense this to if (radio_dec.isChecked) ExportDecrypted else export encrypted